code/modules/spells/spell_types/godhand.dm

```diff
@@ -110,3 +110,46 @@
 	M.Stun(40)
 	M.petrify()
 	return ..()
+
+/obj/item/melee/touch_attack/honk
+	name = "\improper silly touch"
+	desc = "Can you hear it? The pained laughter? The dull smack of a forehead against metal and glass? The incessant need to honk without end?"
+	catchphrase = "Lonk-Lonk-Lonk, FMR'shNK Cluw-NR!!!"
+	on_use_sound = 'sound/items/airhorn.ogg'
+	icon_state = "clown"
+	item_state = "clown"
+
+/obj/item/melee/touch_attack/honk/afterattack(atom/target, mob/living/carbon/user, proximity)
+	if(!proximity || !ishuman(target) || !iscarbon(user))
+		return
+	if(!(user.mobility_flags & MOBILITY_USE))
+		to_chat(user, "<span class='warning'>You can't reach out!</span>")
+		return
+	if(!user.can_speak_vocal())
+		to_chat(user, "<span class='warning'>You can't get the rage out of your system!</span>")
+		return
+	var/mob/living/carbon/human/H = target // Only humans can wear stuff
+	// Check if they already have clown gear on, if so, turn that suit into a clown simplemob
+	var/obj/item/clothing/under/rank/civilian/clown/suit = H.get_item_by_slot(ITEM_SLOT_ICLOTHING)
+	if(istype(suit))
+		H.visible_message("<span class='danger'>[H]'s [suit] animates and jumps off of [H.p_them()], turning into a clown!</span>")
+		H.dropItemToGround(suit)
+		qdel(suit)
+		new /mob/living/simple_animal/hostile/retaliate/clown(H.loc)
```

Comment:
could we make this be a cluwne corpse instead of a living clown? the suit is made of DEAD flesh, after all.

alternatively, add a hidden feature where if you use this on a catatonic person or on a corpse that doesn't have a mind, they turn into a living clown simplemob.

or just ignore me, this isn't a critical feature

Reply:
its a funny interaction like how ei nath reacts poorly to human skinsuits